github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -17,6 +16,7 @@ presubmits:
       preset-podman-in-container-enabled: "true"
     max_concurrency: 1
     name: pull-kubevirtci-bump-kubevirt
+    optional: true
```

Comment:
why is it moved ? automatically ?

Reply:
IIRC that is part of the automation, which when writing back the changed file automatically sorts alphabetically.

Reply:
yes u right, saw it also on https://github.com/kubevirt/project-infra/pull/3126
weird that it is a map, and map on go isnt sorted, but this is what it is